uji.py

```diff
@@ -974,6 +975,16 @@ def _handle_input(self, window, c):
 
         return False
 
+    def _insert(self, offset, line):
+        l = self._sanitize(line)
+        max_len = 250
+        lines = [(l[i:i + max_len]) for i in range(0, len(l), max_len)]
```

Comment:
I admit, having `l`, `line`, `lines` and `self.lines` here makes this more confusing than it needs to be. Moving this into a helper function `line_split_by_width()` or somesuch would help, or at least a comment like this:
```suggestion
        # split our line into max_len-sized chunks, then append them as one
        # line each with trailing newline
        lines = [(l[i:i + max_len]) for i in range(0, len(l), max_len)]
```

Reply:
Used the `line_split_by_width()` suggestion and amended the comments